### DISCUSSION TITLE
Welcome to art-gallery-website Discussions!

### DISCUSSION THREAD

**Discussion by lianaling** [LianaLing/art-gallery-website#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by MariaKruglyak** [comment]
Hello, is the art-gallery-website the software you've built for showcasing art? What's the best use for it? I'm looking for something (and someone!) for an upcoming art journal + exhibition space :)

**Reply by lianaling** [reply]
Hi there :) this website is actually an assignment by our uni, but we don't exclude the possibility of turning it into an open-source software once we've complete developing it.

**Comment by MariaKruglyak** [comment]
Ah, I see! Sick! Do you have a prototype I can take a glance at? I’m working on a digital exhibition space and looking for developers as well as ideas so would love to connect.

**Reply by lianaling** [reply]
I'm afraid we haven't, haha. We're just starting to build the website, and it would take approximately three months (a sem) to complete it. I'm not sure what we can offer you but we can always stay in touch. @marcustut
